trading-rules.adoc

```diff
@@ -4,28 +4,59 @@
 
 _This document is under development. Subscribe to https://github.com/bisq-network/bisq-docs/issues/80[bisq-network/bisq-docs#80] for updates._
 
-To trade on Bisq, you assume one of two roles. You either make a new offer to buy or sell an asset (offer maker), or you agree to accept an existing offer to buy or sell an asset (offer taker).
+To trade on Bisq, you assume one of two roles. You either buy (BTC buyer) or sell (BTC seller) bitcoin.
 
 This document covers the rules and expectations for each role, as well as details on the process for arbitrating disputes.
 
-== Offer maker
+== BTC buyer
 
-_Coming soon._
+* Must make payment from the account used to make/take the offer. The identifiers must match.
+* Must enter the correct reference text in the payment transfer.
+* Must click "Payment started" after having done so.
+* Is responsible for providing the arbitrator with tamper proof evidence of their payment in case of dispute.
+* Must be available, in case of dispute, 48 hours after the max. trade period runs out.
+* Has 48 hours to respond to each arbitrator inquiry.
 
-== Offer taker
+== BTC seller
 
-_Coming soon._
+* Must click "Confirm payment" after having received it. Has till the end of the max. trade period to do so.
+* Must be available, in case of dispute, 48 hours after the max. trade period runs out.
+* Has 48 hours to respond to each arbitrator inquiry.
 
 == Arbitration
 
 When trading partners cannot successfully complete a trade on their own, the trade is disputed, and an arbitrator steps in to determine a solution.
 
-_Coming soon. For now, you can read more about the arbitration process https://bisq.network/faq/#8[here]._
+=== Confirm payment
+
+**The BTC buyer didn't confirm sending the funds and let the trade duration limit run out**
+The arbitrator can release all the escrow funds to the BTC seller.
+
+**The BTC seller didn't release the trade funds by confirming receipt of the funds and let the trade duration limit run out**
+The BTC buyer is responsible to provide the arbitrator with proof of their payment. If they fail to do so and the BTC seller says they haven't received any funds, the BTC seller gets all the funds in escrow.
+The BTC seller has 48 hours after the end of the max. trade period to respond as to whether they have or not received the funds. After that the arbitrator can sign a transaction releasing all funds in escrow to the BTC buyer.
+
+Any effort to reach out to the arbitrators on the Bisq software should be replicated on the link:https://bisq.community/t/arbitrators/8074[forum] as to circumvent network issues.
```

Comment:
@huey735 I missed this before. Do we want to make this a rule? Sounds like overkill. I always assumed people only reach out on the forum only if they have exceptional trouble with the software's messaging tool.

Reply:
Maybe the language can be altered to highlight that it's a suggestion. But recently there has been some issues with the relaying of messages inside Bisq so I think that immediately contacting the arbitrators on the forum is a good preventive measure for now.

Reply:
The problem with users immediately jumping on the forum is that often they display publicly some private infos concerning their trade(s).
And this is a bad practice imo. Which shouldn't be encouraged.

Reply:
@HarryMacfinned maybe the language can be altered to encourage the traders to send private messages on the forum instead?